src/fieldset.c

```diff
@@ -206,6 +206,16 @@ void fs_chkadd_string(fieldset_t *fs, const char *name, char *value, int free_)
 	}
 }
 
+void fs_chkadd_unsafe_string(fieldset_t *fs, const char *name, char *value, int free_)
+{
+	if (value) {
+		fs_add_unsafe_string(fs, name, value, free_);
```

Comment:
This will currently add empty strings. Not sure if that's intended behavior or not.